src/pages/en/guides/deploy/gitlab.md

```diff
@@ -13,30 +13,47 @@ Check out [the official GitLab Pages Astro example project](https://gitlab.com/p
 
 ## How to deploy
 
-1. Set the correct `.site` in `astro.config.mjs`.
-2. Set `dist` in `astro.config.mjs` to `public` and `public` in `astro.config.mjs` to a newly named folder that is holding everything currently in `public`. The reasoning is because `public` is a second source folder in astro, so if you would like to output to `public` you'll need to pull public assets from a different folder. Your `astro.config.mjs` might end up looking like this:
+1. Set the correct `site` in `astro.config.mjs`.
+2. Set `outDir` in `astro.config.mjs` to `public`. GitLab Pages requires exposed files to be located in a folder called "public". This instructs Astro to put the static build output in a folder called `public`.
+
+If you were using the [`public/` directory](/en/reference/configuration-reference/#publicdir) as a source of static files in your Astro project, rename it and use that new folder name in `astro.config.mjs` for the value of `publicDir`.
```

Comment:
```suggestion
2. Set `outDir:public` in `astro.config.mjs`. GitLab Pages requires exposed files to be located in a folder called "public". This setting instructs Astro to put the static build output in a folder called `public`.

If you were using the [`public/` directory](/en/core-concepts/project-structure/#public) as a source of static files in your Astro project, rename it and use that new folder name in `astro.config.mjs` for the value of `publicDir`.
```
Good call on changing the link, and I think we can even do better! I think I like the link to the project structure page where we talk about each main folder/file in a basic Astro project setup.

I also changed the beginning of the instruction to make it less cluttered with all the styled words. Maybe this looks a little cleaner now?

Reply:
I agree that is an even better link. :smile: 

I think the beginning is cleaner now.
But I just now noticed there is two phrases ending with "in a folder called `public`", so maybe if we bring them together the paragraph would look even better.

Maybe:

> This setting instructs Astro to put the static build output in a folder called `public`, which is the folder required by GitLab Pages for exposed files.

Since english is not my first language, I'm not sure if it sounds natural, so I'm totally open for suggestions.

Reply:
I like it a lot! Let's do that, and then I think this PR is good to go!  🥳